rpc/grpc/api.go

```diff
@@ -12,12 +12,12 @@ type broadcastAPI struct {
 	env *core.Environment
 }
 
-func (bapi *broadcastAPI) Ping(ctx context.Context, req *RequestPing) (*ResponsePing, error) {
+func (bapi *broadcastAPI) Ping(context.Context, *RequestPing) (*ResponsePing, error) {
 	// kvstore so we can check if the server is up
 	return &ResponsePing{}, nil
 }
 
-func (bapi *broadcastAPI) BroadcastTx(ctx context.Context, req *RequestBroadcastTx) (*ResponseBroadcastTx, error) {
+func (bapi *broadcastAPI) BroadcastTx(_ context.Context, req *RequestBroadcastTx) (*ResponseBroadcastTx, error) {
```

Comment:
We are setting the variable name to "_" in line 20, and removing the name altogether in line 15?
Can we treat the two cases the same way'

Reply:
The convention in Go is that, if _all_ function parameters are unused, then you can have unnamed parameters. But the moment that one of your parameters is used, they all have to be named, in which case the unused ones must be called `_`.